src/happiness_level.py

```diff
@@ -1,8 +1,8 @@
 import numpy as np
 from typing import Union
-from src.outcomes import Result
```

Comment:
This is incorrect. All imports must be relative import from the root folder of the project. Try to run the notebook `tva.ipynb` it outputs an import error!!